src/types/metricEnum.ts

```diff
@@ -0,0 +1,3 @@
+export enum MetricEnum {
+  MessagesRecieved = "MessagesRecieved",
```

Comment:
Minor spelling mistake.
```suggestion
  MessagesReceived = "MessagesReceived",
```